pioneer/packages/joy-proposals/src/Proposal/Body.tsx

```diff
@@ -81,16 +85,27 @@ const ParsedHRT = styled.pre`
   white-space: pre-wrap;
 `;
 
+type ParsedParamValue = string | number | JSX.Element;
+class FullWidthParam {
+  value: ParsedParamValue;
+  constructor (value: ParsedParamValue) {
+    this.value = value;
+  }
+}
+type ParsedParams = { [paramName: string]: ParsedParamValue | FullWidthParam };
```

Comment:
I wanted to preserve as much simplicity as possible when parsing the paramters (since there's quite a lot of them), so decided to introduce `ParsedParm` object that you can constuct with `new ParsedParam(name, value, fullWidth?)`, I think that makes the code way cleaner than before.